File: C/1012 - Ârea.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    float A, B, C;
 
    scanf("%f %f %f", &A, &B, &C);   

    printf("TRIANGULO: %.3f\n", (A * C)/2);
    printf("CIRCULO: %.3f\n", (C * C) * 3.14159);
    printf("TRAPEZIO: %.3f\n", ((A + B) * C) / 2);
    printf("QUADRADO: %.3f\n", B * B);
    printf("RETANGULO: %.3f\n", A * B);
   
    return 0;
}

File: PYTHON/1012 - Ârea.py
n1, n2, n3 = map(float, input().split(" "))

sum1 = n1 + n2

PA = n1 * n3 /2
PB = n3 * n3 * 3.14159
PC =  sum1 * n3 /2
PD = n2 * n2
PE = n1 * n2

print(f"TRIANGULO: {PA:.3f}")
print(f"CIRCULO: {PB:.3f}")
print(f"TRAPEZIO: {PC:.3f}")
print(f"QUADRADO: {PD:.3f}")
print(f"RETANGULO: {PE:.3f}")